### PR TITLE
Report stale metadata asset

### PR DESCRIPTION
**Pull request by bnchrch**
## Overview
Screenshot 2023-08-01 at 12 10 17 AM

Adds a new report to dagster that shows if files in the github repo are out of date with latest

This is part of #27853 

is waiting on code in https://github.com/airbytehq/airbyte/pull/28759 to complete the slack notification portion

### CODE REVIEW

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py [hunk 1]
```diff
@@ -17,6 +37,59 @@ def github_connector_folders(context):
...
+    metadata_file_paths = {
+        metadata_path: _get_md5_of_github_file(context, github_connector_repo, metadata_path)
+        for metadata_path in github_connectors_metadata_files
+    }
+
+    return Output(metadata_file_paths, metadata={"preview": metadata_file_paths})
+
+
+@asset(required_resource_keys={"latest_metadata_file_blobs"}, group_name=GROUP_NAME)
+def stale_github_metadata_file(context, github_metadata_file_md5s: dict) -> OutputDataFrame:
```
Comment (alafanechere) [inline airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py:57]:
```suggestion
def stale_gcs_latest_metadata_file(context, github_metadata_file_md5s: dict) -> OutputDataFrame:
```
As the source of truth is the github repo, isn't it a "latest" metadata that can be stale?

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py [hunk 2]
```diff
@@ -17,6 +37,59 @@ def github_connector_folders(context):
...
+    """
+    Return a list of all metadata files in the github repo and denote whether they are stale or not.
+
+    Stale means that the file in the github repo is not in the latest metadata file blobs.
+    """
+    latest_metadata_file_blobs = context.resources.latest_metadata_file_blobs
```
Comment (alafanechere) [inline airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py:63]:
```suggestion
    latest_gcs_metadata_file_blobs = context.resources.latest_metadata_file_blobs
```

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py (hunk 1 above)
Comment (alafanechere) [inline airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py:57]: neat function 👏

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/resources/github.py [hunk 3]
```diff
@@ -36,6 +44,22 @@ def github_connectors_directory(resource_context: InitResourceContext) -> List[C
...
 
 
+@resource(
+    required_resource_keys={"github_connector_repo"},
+    config_schema={"connectors_path": StringSource},
+)
+def github_connectors_metadata_files(resource_context: InitResourceContext) -> List[str]:
+    resource_context.log.info(f"retrieving github metadata files")
+
+    github_connector_repo = resource_context.resources.github_connector_repo
```
Comment (alafanechere) [inline airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/resources/github.py:54]: What's the type of this object? Are we interacting we a remote repo by getting raw file content with http requests to Github or are we cloning a the repo locally and read local files?

**Review by alafanechere — APPROVED** [review]
Great straightforward implementation. Just wondiering how the interaction with github files works.

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py (hunk 1 above)
Comment (bnchrch) [reply airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py:57]: good callout!

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/resources/github.py (hunk 3 above)
Comment (bnchrch) [reply airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/resources/github.py:54]: Its the `Repository` type from `pygithub` https://github.com/airbytehq/airbyte/blob/bnchrch/stale-metadata-asset/airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/resources/github.py#L27

Which is all from the github api

**Comment by octavia-squidington-iii** [comment]
##  destination-databend test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ❌

⏲️  Total pipeline duration: 02mn07s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-databend/metadata.yaml | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |
| Code format checks                                                          | ✅        |
| Connector package install                                                   | ✅        |
| Build destination-databend docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ✅        |
| Integration tests                                                           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025315/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-databend/0.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-databend test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-convex test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ✅

⏲️  Total pipeline duration: 02mn00s 

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-convex/metadata.yaml | ✅        |
| Connector version semver check                                            | ✅        |
| QA checks                                                                 | ✅        |
| Code format checks                                                        | ✅        |
| Connector package install                                                 | ✅        |
| Build destination-convex docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                | ✅        |
| Integration tests                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025315/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-convex/0.2.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-convex test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-amazon-sqs test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ✅

⏲️  Total pipeline duration: 02mn21s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-amazon-sqs/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build destination-amazon-sqs docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Integration tests                                                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025315/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-amazon-sqs/0.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-amazon-sqs test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-aws-datalake test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ✅

⏲️  Total pipeline duration: 03mn41s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-aws-datalake/metadata.yaml | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Code format checks                                                              | ✅        |
| Connector package install                                                       | ✅        |
| Build destination-aws-datalake docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                      | ✅        |
| Integration tests                                                               | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025315/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-aws-datalake/0.1.3/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-aws-datalake test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-databend test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ❌

⏲️  Total pipeline duration: 01mn30s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-databend/metadata.yaml | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |
| Code format checks                                                          | ✅        |
| Connector package install                                                   | ✅        |
| Build destination-databend docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ✅        |
| Integration tests                                                           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025974/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-databend/0.1.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-databend test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-convex test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ✅

⏲️  Total pipeline duration: 01mn28s 

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-convex/metadata.yaml | ✅        |
| Connector version semver check                                            | ✅        |
| QA checks                                                                 | ✅        |
| Code format checks                                                        | ✅        |
| Connector package install                                                 | ✅        |
| Build destination-convex docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                | ✅        |
| Integration tests                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025974/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-convex/0.2.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-convex test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-aws-datalake test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ✅

⏲️  Total pipeline duration: 01mn35s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-aws-datalake/metadata.yaml | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Code format checks                                                              | ✅        |
| Connector package install                                                       | ✅        |
| Build destination-aws-datalake docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                      | ✅        |
| Integration tests                                                               | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025974/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-aws-datalake/0.1.3/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-aws-datalake test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-amazon-sqs test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ✅

⏲️  Total pipeline duration: 01mn29s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-amazon-sqs/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build destination-amazon-sqs docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Integration tests                                                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025974/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-amazon-sqs/0.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-amazon-sqs test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-cumulio test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ❌

⏲️  Total pipeline duration: 03mn25s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-cumulio/metadata.yaml | ✅        |
| Connector version semver check                                             | ✅        |
| QA checks                                                                  | ✅        |
| Code format checks                                                         | ❌        |
| Connector package install                                                  | ✅        |
| Build destination-cumulio docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                 | ✅        |
| Integration tests                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025974/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-cumulio/0.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-cumulio test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-duckdb test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ❌

⏲️  Total pipeline duration: 56.68s 

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-duckdb/metadata.yaml | ✅        |
| Connector version semver check                                            | ✅        |
| QA checks                                                                 | ✅        |
| Code format checks                                                        | ✅        |
| Connector package install                                                 | ✅        |
| Build destination-duckdb docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                | ✅        |
| Integration tests                                                         | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025974/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-duckdb/0.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-duckdb test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-firestore test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ✅

⏲️  Total pipeline duration: 01mn10s 

| Step                                                                         | Result   |
|:-----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-firestore/metadata.yaml | ✅        |
| Connector version semver check                                               | ✅        |
| QA checks                                                                    | ✅        |
| Code format checks                                                           | ✅        |
| Connector package install                                                    | ✅        |
| Build destination-firestore docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                   | ✅        |
| Integration tests                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025974/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-firestore/0.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-firestore test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-firebolt test report (commit [`5c643c9dac`](https://github.com/airbytehq/airbyte/pull/28903/commits/5c643c9dac0f01ec9217855442822e1b6f46a2d8)) - ✅

⏲️  Total pipeline duration: 02mn16s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-firebolt/metadata.yaml | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |
| Code format checks                                                          | ✅        |
| Connector package install                                                   | ✅        |
| Build destination-firebolt docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ✅        |
| Integration tests                                                           | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci-internal/connectors/test/pull_request/bnchrch_stale-metadata-asset/1691025974/5c643c9dac0f01ec9217855442822e1b6f46a2d8/destination-firebolt/0.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5c643c9dac0f01ec9217855442822e1b6f46a2d8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-firebolt test
```

**Pull request merged**